STYLE-GUIDE.md

```diff
@@ -2,38 +2,36 @@
 
 This document defines some general styles we adhere to in the docs.
 
-## Words are important
+## Use Inclusive Language
```

Comment:
could we have the words are important line as a line of text below the heading?

Reply:
Yes - with an end to that sentence saying why.  Give me a minute to work on that text.